components/org.wso2.carbon.identity.media.endpoint/src/main/resources/media_endpoint.yaml

```diff
@@ -0,0 +1,525 @@
+openapi: 3.0.0
+info:
+  version: 'v1'
```

Comment:
make the version defined here consistent with the version in the url

Reply:
fixed with:https://github.com/wso2/identity-media/pull/14/commits/9030bd13e5d468797c33c37359cb53bd6de68e4b